src/logging.h

```diff
@@ -75,6 +105,47 @@ namespace BCLog {
         Error = 4,
     };
 
+    static constexpr LogFlags DEFAULT_LOG_FLAGS{UNCONDITIONAL_RATE_LIMITED | UNCONDITIONAL_ALWAYS};
+
+    //! Fixed window rate limiter for logging.
+    class LogRateLimiter
+    {
+    private:
+        //! Timestamp of the last window reset.
+        std::chrono::seconds m_last_reset;
```

Comment:
```suggestion
        std::chrono::steady_clock::time_point m_last_reset{std::chrono::steady_clock::now()};
```

Looks like you are not mocking this in the tests, so could use steady clock?

If not, new code should probably use `NodeClock::now()` over `GetTime()`.

Reply:
Changed it to use `NodeClock`.